### PR TITLE
centos6 python 2.7

### PR DESCRIPTION
**Pull request by jburel**
Add walkthrough for installing omero on cento6 with scl python 2.7

This PR is a subset of the walkthrough generated from omero-install
To review this PR:
- checkout omero-install if you do not have it locally (for convenience) alternatively, you can look at the files under https://github.com/ome/omero-install/tree/develop/linux
- For each step 01-02, 05, check the corresponding file stepXX-centos6-py27-YY.sh
- Step 03-04: check `step03_all_postgres.sh` and step04_all_omero.sh

### CODE REVIEW

**Comment by joshmoore** [comment]
@jburel : can you include the description the steps you're using to generate these?

**Comment by jburel** [comment]
done

File: omero-installation/omero_installation_centos6_sclpython27.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+Postgres 9.4 and reconfigure to allow TCP connections:
+
+	yum -y install postgresql94-server postgresql94
+
+	service postgresql-9.4 initdb
+	sed -i.bak -re 's/^(host.*)ident/\1md5/' /var/lib/pgsql/9.4/data/pg_hba.conf
+	chkconfig postgresql-9.4 on
+	service postgresql-9.4 start
+
+##### Step 2:
```
Comment (pwalczysko) [inline omero-installation/omero_installation_centos6_sclpython27.md:113]: I cannot see the difference in the number of characters here, but evidently this header is considered to be formatted a bit differently than, e.g. Step 3; (which is blue and not shown as italics)

**Comment by pwalczysko** [comment]
step01_centos6_py27_deps.sh contains:

```
# TODO: this installs a lot of unecessary packages:
yum -y groupinstall "Development Tools"

set +u
source /opt/rh/python27/enable
set -u
easy_install pip

export PYTHONWARNINGS="ignore:Unverified HTTPS request"
pip install -r requirements_centos6_py27.txt
```

this part I do not see in the doc.

EDIT: It is in the doc, but in another place, just after Java, Other packages for Ice3.5.

**Comment by pwalczysko** [comment]
In the doc, Java 1.8 chapter is after the python dependencies chapter in step 1.
In the install_centos6_py27_apache24.sh the order is reverse, i.e. first comes the java install, then python deps.

```
# install java
bash -eux step01_centos_java_deps.sh

bash -eux step01_centos6_py27_deps.sh
```

**Comment by jburel** [comment]
The order won't have an impact, the dependencies "Development Tools" are for ice.
So as long as Ice is after.

**Comment by pwalczysko** [comment]
This line
`yum -y install http://yum.postgresql.org/9.4/redhat/rhel-6-x86_64/pgdg-centos94-9.4-2.noarch.rpm` from step01_centos6_pg_deps.sh
Seems to be missing in the doc.

**Comment by pwalczysko** [comment]
`service nginx reload` is in the doc.
In the step05_centos6_py27_nginx.sh there is `service nginx start`

**Comment by pwalczysko** [comment]
```
#If you don't want to use the init.d scripts you can start OMERO manually:
#su - omero -c "OMERO.server/bin/omero admin start"
#su - omero -c "OMERO.server/bin/omero web start"

bash -eux setup_centos_selinux.sh

bash -eux step06_centos6_daemon.sh

bash -eux step07_all_perms.sh

bash -eux step08_all_cron.sh
```

The commented-out part of the previous snippet from install_centos6_py27_nginx.sh is in the doc. The last four bash... commands are not. Possibly this is expected ?

**Comment by pwalczysko** [comment]
In summary, 3 questions. https://github.com/ome/community-docs/pull/3#issuecomment-242343833, https://github.com/ome/community-docs/pull/3#issuecomment-242348332 and https://github.com/ome/community-docs/pull/3#issuecomment-242349923 otherwise the doc has all what is mentioned in install_centos6_py27_nginx.sh.

**Comment by jburel** [comment]
comment https://github.com/ome/community-docs/pull/3#issuecomment-242348332 we have used reload for debian 8, the omero-install script has not been updated

comment: https://github.com/ome/community-docs/pull/3#issuecomment-242343833
seems to be a bug. I will fix it

comment: https://github.com/ome/community-docs/pull/3#issuecomment-242349923 This is expected

**Comment by jburel** [comment]
@pwalczysko: fix pushed

**Comment by pwalczysko** [comment]
The fix in https://github.com/ome/community-docs/pull/3/commits/4a84c687e2a37cc804235e88cd83a65878ba1c39 is as expected.

**Comment by joshmoore** [comment]
i.e. good to merge, @pwalczysko ?

**Comment by pwalczysko** [comment]
Good to merge.

**Comment by joshmoore** [comment]
> done

Sorry, I'm still confused. How is the file created though?

**Comment by jburel** [comment]
It is generating using https://github.com/ome/omero-install/blob/develop/linux/autogenerate.sh

**Comment by joshmoore** [comment]
And you're hand-copying it into the template with "Name: " etc.?

**Comment by jburel** [comment]
You can build only one walkthrough `OS=centos6_py27 ALL=false bash autogenerate.sh`,
or just run `bash autogenerate.sh`` to build all the walkthroughs (but not centos6)

**Comment by joshmoore** [comment]
Sorry, guess I'm not being clear. But the whole thing isn't templated somewhere, right? (i.e. if someone else wanted to do it)

**Comment by jburel** [comment]
we will need to add a new install template in this repo now that we have 2 examples

**Comment by joshmoore** [comment]
Understood.

**Pull request merged**